client/client.lua

```diff
@@ -86,11 +96,18 @@ end)
 
 RegisterNetEvent("QBCore:Client:OnPlayerLoaded", function()
     GetScenes()
+    QBCore.Functions.TriggerCallback('qb-scenes:server:GetPermissions', function(permission)
+        permissionLevel = permission
+    end)
 end)
 
 AddEventHandler('onResourceStart', function(resourceName)
     if resourceName == GetCurrentResourceName() then
+        Wait(2000)
         GetScenes()
+        QBCore.Functions.TriggerCallback('qb-scenes:server:GetPermissions', function(permission)
+            permissionLevel = permission
+        end)
```

Comment:
Since we're using this more than once, we should move this to a function that can be called.